Updating documentation for text_similarity_reranker for 8.x (#126971)
elastic/elasticsearch: docs/reference/reranking/semantic-reranking.asciidoc

. *Select and configure a re-ranking model*.
You have the following options:

> .. Use the <<inference-example-elastic-reranker,Elastic Rerank>> cross-encoder model via a preconfigured `.rerank-v1-elasticsearch` or by creating a custom deployment using the inference API's {es} service.

Reviewer: ```suggestion
.. Use the <<inference-example-elastic-reranker,Elastic Rerank>> model through a preconfigured `.rerank-v1-elasticsearch` endpoint or create a custom one using the inference API's {es} service.
```
